### PR TITLE
Completed javascript-loops exercise. - Jeff Jin

### PR DESCRIPTION
**Pull request by jeffmjin**
Screen Shot 2022-09-01 at 2 45 49 PM

Updated screenshot
Screen Shot 2022-09-01 at 3 13 59 PM

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  for (var i = 0; i < numbers.length; i++) {
+    doubled.push(numbers[i] * 2);
+  }
+  return doubled;
+}
+
+function getKeys(object) {
+  var keys = [];
+  for (var key in object) {
+    keys += key + ',';
```
Comment (BrettAlbright) [inline javascript-loops/main.js]: You want to use the push method here to push the strings into the array.

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  }
+  return keys;
+}
+
+function getValues(object) {
+  var values = [];
+  for (var key in object) {
+    values += object[key] + ',';
```
Comment (BrettAlbright) [inline javascript-loops/main.js]: same as above, you want to use the push method here to add the values to the array

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
a couple small updates!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**